packages/ignite/package.json

```diff
@@ -27,6 +27,7 @@
   "license": "MIT",
   "dependencies": {
     "gluegun": "0.7.3",
+    "gluegun-patching": "^0.3.0",
```

Comment:
@skellock - I tried using this, but saw no way to actually use it, so I just brought in patching first class.  Need to fix this.